app/javascript/admin/controllers/config_controller.js

```diff
@@ -415,20 +415,6 @@ export default class ConfigController extends Controller {
     this.authSectionFormTarget.submit();
   }
 
-  activateMissingKeysModal(providers) {
-    this.configModalAnchorTarget.innerHTML = adminModal({
-      title: 'Getting started',
```

Comment:
Recently you changed [this title from Setup not Complete to Getting Started. ](https://github.com/forem/forem/pull/15969/files#diff-caedbb7d7984a39e9a441e5df7c79b425ec24ebe6d54b4e50261e0634c183793L405)


I don't think that change should have been made as it relates to the setup not complete for the authentication providers not the getting started section.